cmake.deps/cmake/TreesitterCMakeLists.txt

```diff
@@ -9,7 +9,12 @@ add_compile_options(-w)
 
 add_library(tree-sitter lib/src/lib.c)
 target_include_directories(tree-sitter
-  PRIVATE lib/src lib/include)
+  PRIVATE lib/src lib/src/wasm lib/include)
+
+option(ENABLE_WASMTIME "Use treesitter with wasmtime support." OFF)
```

Comment:
@dundargoc is this the right way to do it?

Reply:
This is a bit sus. Why is this needed? It should in theory work without this.

Reply:
Aha, I understand now. Give me a sec.